source/common/quic/envoy_quic_client_session.cc

```diff
@@ -117,22 +93,6 @@ void EnvoyQuicClientSession::OnTlsHandshakeComplete() {
   raiseConnectionEvent(Network::ConnectionEvent::Connected);
```

Comment:
Here I noticed that we are not calling the base class method.
It's not a big issue since the base class method does actual work only on servers. But I think it'd nice to still have those QUIC_BUGs.